### PR TITLE
Implemented task feature, added Junit tests, edited help list and UI

### PR DESCRIPTION
**Pull request by lhydl**

### CODE REVIEW

File: src/main/java/seedu/modtracker/Task.java [hunk 1]
```diff
@@ -43,22 +46,13 @@ public String getStatusIcon() {
         return (isDone ? "✓" : "✘");
```
Comment (limgl1998) [inline src/main/java/seedu/modtracker/Task.java:46]: Not sure if really wanna use this as this is kind of buggy. Might need to update/ tweak cmd for it to show

**Review by limgl1998 — COMMENTED** [review]
Looks good!

File: src/main/java/seedu/modtracker/Task.java (hunk 1 above)
Comment (lhydl) [reply src/main/java/seedu/modtracker/Task.java:46]: Is there a better alternative? I used the above method as using "\u2713" and "\u2718" violates the Gradle style check.

Comment (limgl1998) [reply src/main/java/seedu/modtracker/Task.java:46]: Hmm, then for now we just use it. If it was encountered as a bug during testing then we change? Cos now this method is not tested yet right?

Comment (lhydl) [reply src/main/java/seedu/modtracker/Task.java:46]: Yeah, can.

**Pull request merged**